src/Application/Constants/ContentfulEmailNames.cs

```diff
@@ -0,0 +1,10 @@
+namespace Application.Constants
+{
+    public class ContentfulEmailNames
```

Comment:
Are these constants deployment specific? In which case they should be read from configuration rather than compile time constants.

Reply:
These are not deployment specific

Reply:
They are the id of the contentful entry.  I take your point though as if someone deletes the email then creates a new one it'll need fixed instead of just updating secrets